frontend/src/app/plugins/shape.cljs

```diff
@@ -1309,21 +1309,22 @@
          (cond-> (or (cfh/path-shape? data) (cfh/bool-shape? data))
            (crc/add-properties!
             {:name "content"
-             :get #(-> % u/proxy->shape :content upf/format-path)
+             :get #(-> % u/proxy->shape :content str)
```

Comment:
same here. should we call to `.toString` instead.

Reply:
Fixed